libraries/core-react/src/Accordion/AccordionHeader.tsx

```diff
@@ -72,7 +72,7 @@ const StyledIcon = styled(
   chevronPosition === 'left' ? { marginRight: '32px' } : { marginLeft: '16px' },
 )
 
-type AccordionHeaderTitleProps = Props & AccordionProps
+type AccordionHeaderTitleProps = AccordionHeaderProps & AccordionProps
```

Comment:
This can be simplified as this component does not use all these props (my bad). The following should be enough:
`type AccordionHeaderTitleProps = Pick<
  AccordionHeaderProps,
  'disabled' | 'isExpanded'
>
`